t/ui/28-keys_to_render_as_links.t

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env perl
+# Copyright © 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+
+use FindBin;
+use Test::Mojo;
+use Test::Warnings ':report_warnings';
+use lib "$FindBin::Bin/../lib";
+use OpenQA::Test::Case;
+use OpenQA::SeleniumTest;
+use Mojo::File 'path';
+
+my $schema = OpenQA::Test::Case->new->init_data(fixtures_glob => '01-jobs.pl 05-job_modules.pl');
```

Comment:
I checked the runtime of the test module in https://app.circleci.com/pipelines/github/os-autoinst/openQA/4043/workflows/697d8781-9867-425c-b6e7-45938500ca43/jobs/38525 and it takes 19s which is a lot. I assume that the job module fixtures add a lot of runtime. Do you actually need them?

Reply:
not really. it saves 5s without them. i removed them